src/index.html

```diff
@@ -77,38 +78,6 @@
               app.membershipCredential = {};
               clearOutput();
             },
-
-            discloseEmailAndName: function () {
```

Comment:
Is that dead code?

Reply:
it's not used in that page! we just issue in that page not disclose! Am I right?

Reply:
you are right!